server/src/controllers/attainment.ts

```diff
@@ -371,6 +372,7 @@ export async function getAttainment(req: Request, res: Response): Promise<void>
 }
 
 export async function getRootAttainment(req: Request, res: Response): Promise<void> {
+  const tree: string = await validateTreeParam(req.query.tree as string);
```

Comment:
This does no harm again, but I thought the point was to not include this because it seemed unnecessary. At least that's why I removed it initially.

Reply:
I needed to get easy access to the root attainment and its immediate children without knowing the actual root attainment id.

Reply:
If there turned out to be reason for the feature to exist, then it's fine.